### DISCUSSION TITLE
Images in Maui

### DISCUSSION THREAD

**Discussion by Alef34** [dotnet/maui#12492] [Q&A] [answered]
I have created simple application. I have MainPage and MainPageViewModel.  
On MainPage I have this code

<VerticalStackLayout>

        <Frame BorderColor="Blue" WidthRequest="400" HeightRequest="100" Padding="10">
            <Label Text="{Binding Nadpis}" FontSize="24"/>
        </Frame>
        <Frame BorderColor="Red" WidthRequest="100" HeightRequest="100" Padding="10">
            <Image Source="{Binding Img}" WidthRequest="100" HeightRequest="100"/>
        </Frame>
        <Frame BorderColor="Green" WidthRequest="100" HeightRequest="100" Padding="10">
            <Image Source="lock1.png" WidthRequest="100" HeightRequest="100"/>
        </Frame>

</VerticalStackLayout>

Result is only Label and one image. Bindable Image from ViewModel is not present. 

Where is problem???

Source code is here :
https://github.com/Alef34/ImagesMaui.git

**Comment by mphill** [comment, marked as answer]
Try to use the a type of ImageSource instead of Image.

**Reply by Alef34** [reply]
Hello, it was a great Idea. ImageSource is better option. 
Thanks, Julo
